sys/sys/linker.h

```diff
@@ -67,7 +67,7 @@ struct common_symbol {
     STAILQ_ENTRY(common_symbol) link;
     char*		name;
     caddr_t		address;
-};
+} __aligned(sizeof(uintcap_t));
```

Comment:
In practice that only affects hybrid kernels given it has pointers in it... also should really be `_Alignof(__alignof(uintcap_t))`, or better yet use ALIGNBYTES?

Reply:
Hmm, that's true.  I might just drop this then for now.  Eventually the common symbol stuff should be fixed to get the symbol alignment back from the linker backend and honor that.  I think this change dates from when I was still thinking about making this work for hybrid kernels.